src/services/api/payment.ts

```diff
@@ -25,6 +25,10 @@ const generatePaymentConfiguration = (
       path = `/forms/${form.id}/bpoint-payment`
       break
     }
+    case 'WESTPAC_QUICK_WEB': {
+      path = `forms/${form.id}/westpac-quick-web-payment`
```

Comment:
Also need to add `customerReferenceNumber` to the POST payload. Its similar to the `crn2` and `crn3`